fg/settings/config.py

```diff
@@ -7,18 +7,45 @@
 with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 '''
+import os
+from scif.logger import bot
```

Comment:
If you want to use a logger, django just uses Python logging, here is an example:

https://docs.djangoproject.com/en/2.2/topics/logging/#using-logging

It would also be fine to just use print, or even to remove the silent parameter and skip the verbosity. We don't have an elegant logging system atm :)

Reply:
Ok, I didn't really know how the logging works so I was just conservative there. I'll remove it.